tests/test_rtc_gamma.py

```diff
@@ -93,17 +96,35 @@ def test_golden_tifs(comparison_dirs):
         )
 
         for main_file, develop_file in comparison_files:
-            ret = 0
-            cmd = f'gdalcompare.py {main_file} {develop_file}'
+            comparison_header = '\n'.join(['-'*80, main_file.name, develop_file.name, '-'*80])
             try:
-                stdout = subprocess.check_output(cmd, shell=True, text=True)
-            except subprocess.CalledProcessError as e:
-                stdout = e.output
-                ret = e.returncode
-            print(f'{cmd}\n{stdout}')
-
-            # ret == 0 --> bit-for-bit
-            # ret == 1 --> only binary level differences
-            # ret > 1 -->  "visible data" is not identical
-            # See: https://gdal.org/programs/gdalcompare.html
-            assert ret <= 1
+                compare.bit_for_bit(main_file, develop_file)
+            except compare.ComparisonFailure as b4b_failure:
+                main_ds = xr.open_rasterio(main_file).load()
+                main_ds.close()
+                develop_ds = xr.open_rasterio(develop_file).load()
+                develop_ds.close()
+                try:
+                    xr.testing.assert_identical(main_ds, develop_ds)
+                except AssertionError as identical_failure:
+                    xr_msg = helpers.clarify_xr_message(str(identical_failure))
+                    failure_count += 1
+                    messages.append(f'{comparison_header}\n{xr_msg}')
+
+                    try:
+                        compare.values_are_close(main_ds, develop_ds, atol=0.05)
```

Comment:
Have to set absolute tolerance here, my first area map example differs by up to ~0.04.  Should we set `atol` based on which tiff we're looking at (area vs backscatter)?